src/components/Anchor/Anchor.test.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { ThemeProvider } from "styled-components";
+import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import PropTypes from "prop-types";
+import { assertPropTypes } from "check-prop-types";
+
+import Anchor from "./Anchor";
+import themes from "../common/themes";
+import { fontSizes } from "../common/system";
+
+describe("Component: Anchor", () => {
+
+  const testProps = {
+    className: "my-class",
+    href: "#",
+    style: { color: "red" },
+    children: "foo",
+  }
+
+  it("warns if props are missing or the wrong type", () => {
+    expect(() => { assertPropTypes(Anchor.propTypes, { ...testProps, children: undefined}, 'prop', Anchor.children); }).toThrow();
+    expect(() => { assertPropTypes(Anchor.propTypes, { ...testProps, href: undefined }, 'prop', Anchor.href); }).toThrow();
+    expect(() => { assertPropTypes(Anchor.propTypes, { ...testProps, className: 123 }, 'prop', Anchor.className); }).toThrow();
+    expect(() => { assertPropTypes(Anchor.propTypes, { ...testProps, style: "red" }, 'prop', Anchor.style); }).toThrow();
+  });
+
+  it("renders", () => {
+    const wrapper = shallow(<Anchor />);
+    expect(wrapper.find("Anchor__StyledAnchor").length).toBe(1);
```

Comment:
Can't you also do:
wrapper.find(Anchor) ?
How many elements are even in the wrapper? I feel like you can also just do: expect(wrapper).toHaveLengt(1)

Reply:
expect(wrapper).toHaveLength(1) is cleaner for sure.

Reply:
Yeah, @maracuja-juice is right. Can you fix that Tyler? 😊

Reply:
Sure thing, will do.